q

```diff
@@ -0,0 +1,14 @@
+[core]
```

Comment:
_Hummmmm :grimacing: .._
Maybe you don't want to share your `.git/config` file renamed as a `q` file?